game_model.rb

```diff
@@ -0,0 +1,48 @@
+module GameModel
+  def add_game
+    puts 'Is game multiplayer [Y/N]: '
+    multiplayer = gets.chomp.downcase == 'true'
+    puts 'Enter date of last played(yyyy-mm-dd): '
+    last_played_at = gets.chomp
+    puts 'Enter publish date (yyyy-mm-ddd):'
+    publish_date = gets.chomp
+    puts 'Enter first name: '
+    first_name = gets.chomp
+    puts 'Enter last name: '
+    last_name = gets.chomp
+
+    game = Game.new(multiplayer, last_played_at, publish_date)
+    @games << game
+    # binding pry
+    save_game(@games)
+    Author.new(first_name, last_name)
+
+    puts 'Game  added successfully!'
+  end
+
+  def list_authors
+    authors = []
+    @games.each do |game|
+      authors << game.author unless game.author.nil?
+    end
+
+    if authors.empty?
+      puts 'author: kushie tracy'
+    else
+      authors.uniq.each do |author|
+        puts "First_Name: #{author.first_name} | Last_Name: #{author.last_name}"
+      end
+    end
```

Comment:
As requested by the previous reviewer, I have created a book that required the creation of an author. the new recently created `Author` should be added to the List of  `Authors` 

That's the importance of [Both ways relationships](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)

Please kindly implement that. 💪

Reply:
Thanks for the comment